package.json

```diff
@@ -56,6 +56,7 @@
         "jest": "^29.1.2",
         "pre-commit": "^1.2.2",
         "prettier": "^2.6.1",
+        "query-string": "^8.1.0",
```

Comment:
URL constructor also has a nice api to interact with query string
would be cool to use that, instead of adding a new dep

Reply:
I not familiar with that - will google it.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URL
its a pretty cool Web API

Reply:
removed